code/modules/mob/living/simple_animal/hostile/mining_mobs/gutlunch.dm

```diff
@@ -123,6 +123,7 @@
 //Gutlunch udder
 /obj/item/udder/gutlunch
 	name = "nutrient sac"
+	milktype = "milk"
```

Comment:
The gutlunch makes two different kinds of reagents, the proc is still a hardcoded one so gotta figure out that one something (pickweight from list?)

Reply:
Couldn't I make a generic milk type for different alien critters, if you only want different reagents and not a milk?

Reply:
Actually, I'm going to make everything a list so it looks cleaner.